ui/app/helpers/utils/fetch-with-cache.js

```diff
@@ -2,12 +2,17 @@ import {
   loadLocalStorageData,
   saveLocalStorageData,
 } from '../../../lib/local-storage-helpers'
+import http from './fetch'
+
+const fetch = http({
+  timeout: 1000,
```

Comment:
Why is this time so short?

Reply:
Would this disrupted slower internet connections, or even vpn connections?

Reply:
What's a good value? I assume we want something less than the default 2 minutes?

Reply:
I'll do 30 seconds.